README.md

```diff
@@ -41,3 +41,20 @@ https://example-jenkins-jenkins-demo.apps.my-cluster.testcluster.openshift.com:4
 And then, navigate to the route in your browser. You will be redirected
 by Jenkins to log into the OpenShift console before the Jenkins console
 web UI is opened.
+
+## Creating a Pipeline
+
+Creating a new project that defines a pipeline (for example, through a
+project's JenkinsFile) will automatically create a job and build in
+the Operator managed instance of Jenkins.
+
+For example:
+``` bash
+oc new-app https://github.com/waveywaves/nodejs-ex --context-dir openshift/pipelines
```

Comment:
Interesting, does this really use the operator managed jenkins?

Reply:
this uses any installed jenkins with the service name "jenkins". The readme update is useful for users who want to simply verify that the Jenkins is up and running and runnable.